.gitignore

```diff
@@ -1,7 +1,62 @@
-.DS_Store
-.dart_tool/
+# Created by https://www.toptal.com/developers/gitignore/api/flutter
+# Edit at https://www.toptal.com/developers/gitignore?templates=flutter
```

Comment:
We probably want this to be 

https://www.toptal.com/developers/gitignore?templates=dart,swift,kotlin,flutter,intellij,androidstudio,visualstudiocode,osx,windows

Reply:
Nice!